packages/retail-ui/components/DropdownContainer/DropdownContainer.tsx

```diff
@@ -57,12 +57,14 @@ export default class DropdownContainer extends React.Component<DropdownContainer
   }
 
   public componentWillMount() {
-    const htmlPosition = getComputedStyle(document.documentElement).position;
-    const bodyPosition = getComputedStyle(document.body).position;
+    const { body, documentElement: docEl } = document;
+    const htmlPosition = getComputedStyle(docEl).position;
+    const bodyPosition = getComputedStyle(body).position;
 
-    if (htmlPosition !== 'static' || bodyPosition !== 'static') {
-      this.setState({ hasStaticRoot: false });
-    }
+    const hasLimitedHeightRoot = docEl.offsetHeight !== docEl.scrollHeight;
```

Comment:
```suggestion
    const hasLimitedHeightRoot = body.scrollHeight > body.clientHeight;
```

Пришли к такому решению.